glide.lock

```diff
@@ -1,26 +1,19 @@
-hash: 23db89af71e3e11cfa7bb7722f8dd338f3285dd87270f8e6000898662e7def08
-updated: 2017-04-03T13:42:53.811824499-07:00
+hash: 6829e27eafddc4c5cbd00da491ddbcb8a6d330b7034bee743a3b9abec260f9f9
```

Comment:
i'm assuming we'll also test with 1.5 as well until 1.6 is officially our way forward